### PR TITLE
dconf*: add/refresh pages

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** latest

### CODE REVIEW

File: pages/linux/dconf-read.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# dconf read
+
+> Read key values from dconf databases.
+> See also: `dconf`.
+> More information: <https://manned.org/dconf>.
+
+- Print a specific key value:
+
+`dconf read {{/path/to/key}}`
```
Comment (blueskyson) [inline pages/linux/dconf-read.md:9]:
```suggestion
`dconf read {{path/to/key}}`
```

Comment (EmilySeville7cfg) [reply pages/linux/dconf-read.md:9]: Without leading slash I obtain: `error: dconf key must begin with a slash` for smtl like `dconf read org/mate/panel/toplevels/top/background/color`.

Comment (blueskyson) [reply pages/linux/dconf-read.md:9]: Oh! Got it.

**Review by blueskyson — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
@navarroaxel, @CleanMachine1, please review this PR. 😄

**Review by kbdharun — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Thanks to myself! ❤️

**Comment by kbdharun** [comment]
> Thanks to myself! ❤️

😆 , Great work, Thanks

**Pull request merged**